includes/results.inc

```diff
@@ -550,7 +550,28 @@ class IslandoraSolrResults {
         $format = $this->dateFormatFacets[$solr_field]['solr_field_settings']['date_facet_format'];
         $filter_split[1] = format_date(strtotime(stripslashes($filter_split[1])), 'custom', $format);
       }
-      $filter_string = $filter_split[1];
+      $facet_fields_settings = islandora_solr_get_fields('facet_fields', TRUE, FALSE, TRUE);
+      $filter_string = str_replace('info:fedora/', '', stripslashes($filter_split[1]));
+      if (isset($facet_fields_settings[$solr_field]['solr_field_settings']['pid_object_label']) && $facet_fields_settings[$solr_field]['solr_field_settings']['pid_object_label'] == 1) {
```

Comment:
Is this faster than loading the islandora object and getting the label? Just wondering if that single row query could be an overkill?

Reply:
Not sure, I was using the existing translate for the facets as the source for this work. I'm fine just loading the object to get the label.

Reply:
Oh, cool. So if this is done like this somewhere else, who am i to say different? 😄 Other committers? I have no strong opinion really

Reply:
Elsewhere the solr query could contain all the facets at once though. So it might be more efficient to just load the object from Fedora to get the label. I'm not sure if a single row Solr query is faster or slower than loading an object from Fedora.

Reply:
This piece here https://github.com/Islandora/islandora_solr_search/blob/bcb6301255c297101e43ef6b7110e3362c1a71b2/includes/results.inc#L1001 says lets use islandora_object_load, but could be also based on legacy decisions